File: service/src/main/java/com/service/config/AppProperties.java
package com.service.config;

import lombok.Data;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;
import org.springframework.stereotype.Component;


//@Configuration
//@ConfigurationProperties(prefix = "app")
@Component
public class AppProperties {
    @Value("${ecos-api-key}")
    private  String ecosApiKey;

    public String getEcosApiKey(){
        return this.ecosApiKey;
    }
    public void setEcosApiKey(@Value("${ecos-api-key}") String ecosApiKey) {
        this.ecosApiKey = ecosApiKey;
    }
}

File: src/main/kotlin/com/minssan9/financial/config/AppProperties.kt
package com.minssan9.financial.config

import org.springframework.beans.factory.annotation.Value
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties("account")
class AppProperties {

//    @Value("\${krbankkey}")
    val krbankkey: String = "A3KS6JOGXDYWDBC2XQ3Y"
}
